en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -21,60 +21,59 @@ Follow the procedure below to create an API using the mock REST service you test
 
 4. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
 5. In the **Create API from REST API** form, fill in the details as follows:
-    - **API Name**: `HelloWorld`
-    - **API Version**: `1.0.0`
-    - **Endpoint**: http://www.mocky.io/v2/5185415ba171ea3a00704eed
+    - **API Name**: **HelloWorld**
+    - **API Version**: **1.0.0**
+    - **Endpoint**: **http://www.mocky.io/v2/5185415ba171ea3a00704eed**
 6. Click **Create**. This displays an overview of the API.
 
-       ![API Overview](../assets/img/apis/api-overview.png)
+      ![API Overview](../assets/img/apis/api-overview.png)
        
-7. Under **API Configuration**, click **Resources**.
-8. Follow the procedure below to add a resource to the API:
-    1. Select `GET` as the HTTP verb.
-    2. Enter `hello` as the URI pattern.
+7. Under **API Configuration**, click **Resources**. Then follow this procedure to add a resource to the API:
+    1. Select **GET** as the HTTP verb.
+    2. Enter **hello** as the URI pattern.
     3. Click the **+** icon to add the resource.
-    4. Click **Save**. You can see a confirmation message to proceed with the changes.
+    4. Click **Save**. You'll see a confirmation message to proceed with the changes.
```

Comment:
Is the contraction `'ll` appropriate for formal writing?

Reply:
We have avoid this approach generally but since the copy editor had suggested this throughout most content, I checked a few microsoft style-based docs and found similar usage [1]. Checked with the Choreo writers and we thought of accepting the suggestion. 
[1] https://docs.microsoft.com/en-us/microsoftteams/troubleshoot/meetings/how-to-join-a-teams-meeting#join-with-a-computer-or-laptop
We can discuss all such concerns and finalize guidelines on how we wan't to proceed.